src/plugins/data_source/server/plugin.ts

```diff
@@ -81,6 +83,26 @@ export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourc
       dataSourceSavedObjectsClientWrapper.wrapperFactory
     );
 
+    const { editMode } = config;
+    core.capabilities.registerProvider(() => ({
+      dataSource: {
+        canEdit: false,
+        canAssign: false,
```

Comment:
Do we need `canAssign`? seems it's not used

Reply:
The 'canAssign' is reserved for @raintygao.

Reply:
I don't think we should maintain this flag in data source plugin, it sounds better to be moved to workspace plugin.